hs_core/views/__init__.py

```diff
@@ -451,13 +451,13 @@ def my_resources(request, page):
 #    if not request.user.is_authenticated():
 #        return HttpResponseRedirect('/accounts/login/')
 
-    #import sys
-    #sys.path.append("/home/docker/pycharm-debug")
-    #import pydevd
+    import sys
+    sys.path.append("/home/docker/pycharm-debug")
+    import pydevd
```

Comment:
It seems dangerous to keep this kind of thing in a place that can be accidentally merged and slip into a staging or production environment.